manifests/config.pp

```diff
@@ -69,7 +73,42 @@
     purge   => $purge,
     recurse => $recurse,
   }
+ 
+  if $collectd_selinux and $::selinux == 'true' {
```

Comment:
::selinux is a boolean and I'd expect comparing to a string will fail on Puppet4.